main.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+# Module: plugin.video.magaritv
+# Version: 1.0.0
+# Author: Orazio Giliberti
+# Created on: 29.04.2016
+# License: GPL v.3 https://www.gnu.org/copyleft/gpl.html
+# This module was a modificated version of
+# https://github.com/romanvm/plugin.video.example/releases
+# released by Roman V M under License: GPL v.3 https://www.gnu.org/copyleft/gpl.html
+# Thanks Roman !
+
+
+import sys
+import json
+from urlparse import parse_qsl
+import xbmcgui
+import xbmcplugin
+
+
+__url__ = sys.argv[0]
+__handle__ = int(sys.argv[1])
+
+url = "http://serviceone.magari.tv/video_json2.html"
+
+try:
+    # For Python 3.0 and later
+    import urllib
+    from urllib.request import urlopen
+    response = urlopen(url)
+
+except ImportError:
+    # Fall back to Python 2's urllib2
+    import urllib2
+    from urllib2 import urlopen
+    response = urllib2.urlopen(url)
+
+print(sys.version[0])
+
+string = response.read().decode('utf-8')
+VIDEOS = json.loads(string)
+
+def get_categories():
+    """
+    Get the list of video categories.
+    Here you can insert some parsing code that retrieves
+    the list of video categories (e.g. 'Movies', 'TV-shows', 'Documentaries' etc.)
+    from some site or server.
+    :return: list
+    """
+    return VIDEOS.keys()
+
+
+def get_videos(category):
+    """
+    Get the list of videofiles/streams.
+    Here you can insert some parsing code that retrieves
+    the list of videostreams in a given category from some site or server.
+    :param category: str
+    :return: list
+    """
+    return VIDEOS[category]
+
+
+def list_categories():
+    """
+    Create the list of video categories in the Kodi interface.
+    :return: None
+    """
+    categories = get_categories()
+    listing = []
+    for category in categories:
+        list_item = xbmcgui.ListItem(label=category, thumbnailImage=VIDEOS[category][0]['thumb'])
+        list_item.setProperty('fanart_image', VIDEOS[category][0]['thumb'])
```

Comment:
May you be more precise.
What is the statement I should modify ?

Thanks for your attention